john_pham/Assignments/OOP/bike.py

```diff
@@ -0,0 +1,24 @@
+class Bike(object):
+    def __init__(self, price, max_speed, miles = 0):
+        self.price = price
+        self.max_speed = max_speed
+        self.miles = 0
```

Comment:
Could change this line to: `self.miles = miles` since you are already setting `miles = 0` in the __init__ method's arguments.